sdk/translation/Azure.AI.Translation.Document/tests/GetAllTranslationsFilterTests.cs

```diff
@@ -144,6 +144,10 @@ public async Task GetTranslationStatusesFilterByCreatedAfterTest()
             // create test jobs
             await CreateTranslationJobsAsync(client, jobsCount: 1, docsPerJob: 1, jobTerminalStatus: DocumentTranslationStatus.Succeeded);
             var timestamp = Recording.UtcNow;
+
+            // Wait to account for any time discrepancies
+            Thread.Sleep(1000);
```

Comment:
I'm not sure that we need to sleep here.  Could we not just subtract a second from `timestamp` to induce the same behavior artificially?

Reply:
Yes that does seem better, thanks.

Since we don't assert anything about the job created in the beginning, this should effectively be the same.